src/backend/src/prisma/migrations/20241122155216_updated_notifications/migration.sql

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
did you make the original notification sql file? if so consolidate the two into one please